src/game/Editor/Sector_Summary.cc

```diff
@@ -2199,8 +2194,7 @@ static void RegenerateSummaryInfoForAllOutdatedMaps(void)
 	for (sMap.y = 1; sMap.y <= 16; sMap.y++) for(sMap.x = 1; sMap.x <= 16; sMap.x++)
 	{
 		INT32 x = sMap.x - 1, y = sMap.y - 1;
-		char str[40];
-		sprintf(str, "%s", sMap.AsShortString().c_str());
+		const ST::string& str = sMap.AsShortString();
```

Comment:
Without the &, please. The compiler may be able to optimize this away, but in theory you're creating a temporary string and a reference here.